### PR TITLE
feature/cambiosInstoreGenerales

### PR DESCRIPTION
**Pull request by juanseferrari2**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/in-person-payments/qr-code/introduction.pt.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Para receber pagamentos com QR em seu negócio, **você pode imprimi-lo e colá-
 >
 > Nota
 >
-> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [gerar QR do painel](https://www.mercadopago.com.ar/developers/pt/guides/qr-code/integrations-front/).
+> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [gerar QR do pagina web do Mercado Pago](https://www.mercadopago.com.br/developers/pt/guides/in-person-payments/qr-code/integrations-front/).
```
Comment (lamaralmp) [inline guides/in-person-payments/qr-code/introduction.pt.md:50]:
```suggestion
> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [geração de QR pelo painel do Mercado Pago](https://www.mercadopago.com.br/developers/pt/guides/in-person-payments/qr-code/integrations-front/).
```

File: guides/in-person-payments/qr-code/introduction.en.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ In order to receive payments with QR, you can **print it and paste it in a visib
 >
 > Note
 >
-> If your model does not require integration with a point of sale system, you can consult all the information available in [generate QR from the panel](https://www.mercadopago.com.ar/developers/en/guides/qr-code/integrations-front/).
+> If your model does not require integration with a point of sale system, you can consult all the information available in [generate QR from the Mercado Pago web page](https://www.mercadopago.com.ar/developers/en/guides/in-person-payments/qr-code/integrations-front/).
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/introduction.en.md:49]:
```suggestion
> If your model does not require integration with a point of sale system, you can consult all the information available in [generate QR from the Mercado Pago panel](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/in-person-payments/qr-code/integrations-front).
```

File: guides/in-person-payments/qr-code/introduction.es.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Para recibir pagos con QR en tu negocio **puedes imprimirlo y pegarlo en un luga
 > 
 > Nota
 > 
-> Si tu modelo no requiere de una integración con un sistema de punto de venta,  puedes consultar toda la información disponible en [generar QR desde el panel](https://www.mercadopago.com.ar/developers/es/guides/qr-code/integrations-front/). 
+> Si tu modelo no requiere de una integración con un sistema de punto de venta,  puedes consultar toda la información disponible en [generar QR desde la página de Mercado Pago](https://www.mercadopago.com.ar/developers/es/guides/in-person-payments/qr-code/integrations-front/).
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/introduction.es.md:49]:
```suggestion
> Si tu modelo no requiere de una integración con un sistema de punto de venta,  puedes consultar toda la información disponible en [generar QR desde el panel de Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/in-person-payments/qr-code/integrations-front). 
```

File: guides/in-person-payments/qr-code/introduction.pt.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ Para receber pagamentos com QR em seu negócio, **você pode imprimi-lo e colá-
 >
 > Nota
 >
-> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [gerar QR do painel](https://www.mercadopago.com.ar/developers/pt/guides/qr-code/integrations-front/).
+> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [geração de QR pelo painel do Mercado Pago](https://www.mercadopago.com.br/developers/pt/guides/in-person-payments/qr-code/integrations-front/).
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/introduction.pt.md:50]:
```suggestion
> Caso seu modelo não necessite de integração com sistema de ponto de venda, você pode consultar todas as informações disponíveis em [geração de QR pelo painel do Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/in-person-payments/qr-code/integrations-front).
```

File: guides/in-person-payments/qr-code/qr-attended-part-b.es.md [hunk 5]
```diff
@@ -24,6 +24,12 @@ Te explicamos cómo funciona el modelo atendido:
 1. El punto de venta registra un pedido (1a) y crea una orden asignada a una caja (1b). En este momento la orden se encuentra disponible para ser escaneada (2). 
 2. Cuando el cliente escanea el QR (3) con la orden y realiza el pago (5), se recibe una notificación IPN (4a y 6b) al servidor del vendedor. Con esos datos, se obtiene el estado de la orden (7a), para validar que esté cerrada o siga abierta, pendiente de pago.
 
+## Video explicativo
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/qr-attended-part-b.es.md:27]: Quitar

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**